code/modules/cargo/packs/_packs.dm

```diff
@@ -73,6 +73,8 @@
 			contains[item] = 1
 		for(var/iteration = 1 to contains[item])
 			var/atom/A = new item(C)
+			if(contraband)
```

Comment:
Does this hit the contents of these items?

It is possible to get syndicate smokes from the uplink and the contraband cargo pack. If this doesn't apply the trait to the contents of storage containers, that would mean contraband cigarettes via cargo would avoid the trait, but the uplink ones wouldn't.